src/website/src/app/documentation/demos/button-group/button-group.demo.html

```diff
@@ -185,16 +185,16 @@ <h5>Width & Spacing</h5>
                             <div class="clrweb-DoxMedia-img">
                                 <div class="btn-group btn-primary btn-icon">
                                     <button class="btn">
-                                        <clr-icon shape="plus"></clr-icon>
-                                        <span class="clr-icon-title">Add</span>
+                                        <clr-icon shape="check" title="check"></clr-icon>
```

Comment:
Not sure if I'm doing it wrong (the test).. 

```html
 <div class="btn-group btn-primary btn-icon">
       <button class="btn">
              <clr-icon shape="check" title="Check"></clr-icon>
              <span class="clr-icon-title">Check</span>
      </button>
      ....
```

![Kapture 2019-07-05 at 15 24 38](https://user-images.githubusercontent.com/204564/60722361-589ac100-9f39-11e9-809a-e3c145bf2987.gif)

When removed `title="Check"` the reading was only "button" - Not sure why the screen reader ignore the `<span class="clr-icon-title">Check</span>`


This is working fine..
```html
 <button class="btn">
     User Profile
     <clr-icon shape="user"></clr-icon>
</button>


 <button class="btn">
     <clr-icon shape="user"></clr-icon>
     User Profile
</button>
```

Reply:
I think this is because the `clr-icon-title` uses display none to hide the text and the screen reader won't pick it up because of this. https://github.com/vmware/clarity/blob/master/src/clr-angular/button/button-group/_button-group.clarity.scss#L128 I don't think we want to use the `.clr-icon-title` anymore but either use the `title` on the icon element or `aria-label` on the button as described here https://clarity.design/icons/accessibility

Reply:
The reason I think the `clr-icon-title` exists in those examples was it was to trying to make it accessible but it was before we knew to properly use the `aria-label` or the `title` input on the icon.

Reply:
It looks like in the docs the `clr-icon-title` is only used for the overflow group 

![Screen Shot 2019-07-05 at 8 52 38 AM](https://user-images.githubusercontent.com/2021067/60726932-579a6d00-9f02-11e9-9562-2b33e4797ed3.png)